spec/tree_spec.rb

```diff
@@ -1,14 +1,73 @@
 require 'rspec'
 require 'tree'
 
-describe 'Tree' do
+describe Tree do
   it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+    expect(described_class.is_a? Class).to be_truthy # for some reason expect(true).to be_true is failing
```

Comment:
Try to be more specific. Checking that it's a class is great, but their's another check you should do other than this. (Hint initialize one and see what class it is).